edx/edx-enterprise
# ENT-615 Discount Text on Course Enrollment Landing Page

**Description:** Whenever the enterprise provides a discount to learners, that should be indicated in close proximity to the discount. 

@brittneyexline @zubair-arbi @douglashall can u take a look on this small change. 

<img width="1249" alt="screen shot 2017-09-05 at 2 33 49 pm" src="https://user-images.githubusercontent.com/7334669/30055512-2c49c37a-9249-11e7-90aa-0abe6718878f.png">

<img width="1274" alt="screen shot 2017-09-06 at 3 15 39 pm" src="https://user-images.githubusercontent.com/7334669/30107125-468f9d1c-9317-11e7-87a9-6669763706c7.png">



**JIRA:** [ENT-615](https://openedx.atlassian.net/browse/ENT-615)

**Testing instructions:**

1. Apply a discount to an enterprise verified course.
2. Visit the enterprise enrollment landing page.
3. See the text "Discount provided by [Enterprise]" below the strike value as shown in the attachment.



**Post merge:**
